module/pi_cam.py

```diff
@@ -0,0 +1,127 @@
+from picamera import PiCamera
+import time
+from time import strftime, localtime
+from fractions import Fraction
+
+
+class PiCam:
+    """
+    Picam class
+    Default resolution = 1280 by 720
+    """
+    def __init__(self):
+        self.camera = PiCamera()
+        self.camera.resolution = (1280, 720)
+        time.sleep(2)
```

Comment:
is this sleep here necessary

Reply:
According to the documentation it does need a little bit of time to startup.

Reply:
alright 👍